projects/ngrx.io/content/guide/store/actions.md

```diff
@@ -6,6 +6,23 @@ Actions are one of the main building blocks in NgRx. Actions express _unique eve
 
 Actions are used in many areas of NgRx. Actions are the inputs and outputs of many systems in NgRx. Actions help you to understand how events are handled in your application. This guide provides general rules and examples for writing actions in your application.
 
+## An action is an object
```

Comment:
How is this different from the `Action interface` section below? They basically say the same thing.

Reply:
hi Brandon :) So my idea was to mention this as early as possible. My general idea with the docs is to present the vanilla choice as early as possible and then show in the next section what benefits it holds to use interfaces and classes.. I feel a lot of the confusion about NGRX comes from people not understanding what happens in Vanilla, their lack of typescript and then they get confused